Add --quiet-breakpoints command to gdb-4.16
==========================================

Patch to add --quiet-breakpoints command option which is stops all those
annoying "Breakpoint ..." setting messages. 

Warning: we use a global variable to communicate between the main.c
and the mention() function, obviously a bad idea and we should do it
properly (which is....)

Phil Maker (pjm@gnu.org)

cvs rdiff: Diffing gdb/gdb
Index: gdb/gdb/breakpoint.c
diff -c gdb/gdb/breakpoint.c:1.1.1.1 gdb/gdb/breakpoint.c:1.2
*** gdb/gdb/breakpoint.c:1.1.1.1	Wed May 29 17:43:26 1996
--- gdb/gdb/breakpoint.c	Thu May 30 09:29:39 1996
***************
*** 2204,2209 ****
--- 2204,2213 ----
  {
    int say_where = 0;
  
+   extern int quiet_breakpoints ;
+   if(quiet_breakpoints) /* command line flag to disable message (pjm) */
+        return;
+   
    /* FIXME: This is misplaced; mention() is called by things (like hitting a
       watchpoint) other than breakpoint creation.  It should be possible to
       clean this up and at the same time replace the random calls to
Index: gdb/gdb/main.c
diff -c gdb/gdb/main.c:1.1.1.1 gdb/gdb/main.c:1.2
*** gdb/gdb/main.c:1.1.1.1	Wed May 29 17:43:38 1996
--- gdb/gdb/main.c	Thu May 30 09:29:43 1996
***************
*** 62,67 ****
--- 62,70 ----
  
  extern void gdb_init PARAMS ((void));
  
+ /* used to quieten breakpoint setting messages with --quiet-breakpoints */
+ int quiet_breakpoints = 0;
+ 
  int
  main (argc, argv)
       int argc;
***************
*** 154,159 ****
--- 157,163 ----
  	{"m", no_argument, &mapped_symbol_files, 1},
  	{"quiet", no_argument, &quiet, 1},
  	{"q", no_argument, &quiet, 1},
+ 	{"quiet-breakpoints", no_argument, &quiet_breakpoints, 1}, /* pjm */
  	{"silent", no_argument, &quiet, 1},
  	{"nx", no_argument, &inhibit_gdbinit, 1},
  	{"n", no_argument, &inhibit_gdbinit, 1},
cvs rdiff: Diffing gdb/gdb/29k-share
cvs rdiff: Diffing gdb/gdb/29k-share/udi
cvs rdiff: Diffing gdb/gdb/config
cvs rdiff: Diffing gdb/gdb/config/a29k
cvs rdiff: Diffing gdb/gdb/config/alpha
cvs rdiff: Diffing gdb/gdb/config/arm
cvs rdiff: Diffing gdb/gdb/config/convex
cvs rdiff: Diffing gdb/gdb/config/gould
cvs rdiff: Diffing gdb/gdb/config/h8300
cvs rdiff: Diffing gdb/gdb/config/h8500
cvs rdiff: Diffing gdb/gdb/config/i386
cvs rdiff: Diffing gdb/gdb/config/i960
cvs rdiff: Diffing gdb/gdb/config/m68k
cvs rdiff: Diffing gdb/gdb/config/m88k
cvs rdiff: Diffing gdb/gdb/config/mips
cvs rdiff: Diffing gdb/gdb/config/none
cvs rdiff: Diffing gdb/gdb/config/ns32k
cvs rdiff: Diffing gdb/gdb/config/pa
cvs rdiff: Diffing gdb/gdb/config/powerpc
cvs rdiff: Diffing gdb/gdb/config/pyr
cvs rdiff: Diffing gdb/gdb/config/romp
cvs rdiff: Diffing gdb/gdb/config/rs6000
cvs rdiff: Diffing gdb/gdb/config/sh
cvs rdiff: Diffing gdb/gdb/config/sparc
cvs rdiff: Diffing gdb/gdb/config/tahoe
cvs rdiff: Diffing gdb/gdb/config/vax
cvs rdiff: Diffing gdb/gdb/config/w65
cvs rdiff: Diffing gdb/gdb/config/z8k
cvs rdiff: Diffing gdb/gdb/doc
cvs rdiff: Diffing gdb/gdb/gdbserver
cvs rdiff: Diffing gdb/gdb/nindy-share
cvs rdiff: Diffing gdb/gdb/nlm
cvs rdiff: Diffing gdb/gdb/vx-share
